generator/.pulp-cache/build-stamp

```diff
@@ -0,0 +1 @@
+7b2482593750e1d83b2fb300c2ab41a9
```

Comment:
Actually this isn't I have no idea what this is. Thanks for spotting it *removes*